data/korath/korath missions.txt

```diff
@@ -61,3 +61,353 @@ mission "Discovered Korath Space"
+			choice
+				`	"To be honest, I'm not sure. A Hai named Eruk gave me it; he might have an answer. That would be a good place to start."`
+				`	"Sorry, I don't know, and I'm not able to help right now. I'll return when I'm available. Good luck."`
+					decline
+			`	"A place to start is all we need. Let me tell Syciil," Rek says, then turns to her and repeats your answer. When she's done, Syciil holds her hands up with her palms toward you and moves them in a circular motion while speaking. "That's a gesture of gratitude. She says, 'Thank you. Your help will benefit both the Wanderers and Kor Efreti.' We would go with you, but both of us are busy with another project right now. Please return to Setar Fort if you make any progress." Rek and Scyiil depart. It sounds like you should visit Eruk on <planet>.`
+				accept
+
+mission "Translator Development 1B"
+	landing
+	name "Translator Development"
+	description "Go to <destination> to ask Eruk if his translation device could be modified for the Wanderers and Korath to use."
+	minor
+	source "Setar Fort"
+	destination "Greenwater"
+	to offer
+		has "Wanderers: Sestor: Final: done"
+		has "Translator Development 1A: declined"
+	on offer
+		conversation
+			`As you leave your ship, you remember the conversation you had with Rek and Syciil. They asked you to go speak to Eruk on <destination> about using his translator for the Wanderers and Kor Efreti, but you were too busy to help them. Would you like to tell Rek you can help now?`
+			choice
+				`	(Yes.)`
+				`	(No.)`
+					defer
+			`	You send a message to Rek to let her know you'll go ask Eruk about his translator. She thanks you and says to return to Setar Fort if you make any progress. Time to visit Eruk.`
+				accept
+		
+mission "Translator Development 2A"
+	landing
+	name "Information Exchange"
+	description "Acquire a copy of the Library of Parliament on <stopovers> for Eruk. Then, return to <destination> in exchange for information about the inner workings of his translation device."
+	source "Greenwater"
+	stopover "Earth"
+	to offer
+		or
+			has "Translator Development 1A: done"
+			has "Translator Development 1B: done"
+	on offer
+		conversation
+			`You make contact with Eruk soon after landing, and he invites you on a tour of his facility. Compared to the last time you were here, it has doubled - maybe even tripled - in size. "The translator was released to the public a few months ago, and its reception has been fantastic," he explains. "Consequentially, my operation here has grown immensely." As the tour continues, he's sure to point out many exact details of his work. Eruk is obviously pleased with the success of his device. "But, enough about me. I suspect you're a busy person, so you likely came here with a purpose. What can I do for you?"`
```

Comment:
Facility? Also the whole thing about releasing the translator to the public seems a bit iffy, especially when looking at Sayari's comment about the translation machines:
`She says, "There are scientists on the Hai world of <planet> who develop speaking machines, to allow the Hai who do not know the human language to communicate with humans. I will write a letter asking them to give you one of these speaking machines, so that you may talk with the Wanderers without needing me as an intermediary."`
From this, it sounds like the machine is already public and used, as well as Eruk not being the sole creator of translators, and is just part of a team.

Reply:
Rewritten.